src/applications/gi/containers/ProfilePage.jsx

```diff
@@ -103,21 +104,22 @@ export class ProfilePage extends React.Component {
                     />
                   </AccordionItem>
                 )}
-                {!isOJT && (
-                  <AccordionItem button="Student outcomes">
-                    <If
-                      condition={
-                        !!profile.attributes.facilityCode && !!constants
-                      }
-                      comment="TODO"
-                    >
-                      <Outcomes
-                        graphing={outcomes}
-                        onShowModal={this.props.showModal}
-                      />
-                    </If>
-                  </AccordionItem>
-                )}
+                {!isOJT ||
```

Comment:
Should this be below so it shows if it is not OJT and it is production environment?
```suggestion
                {!isOJT && environment.isProduction() && (
```

Reply:
Changed!